extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.js

```diff
@@ -374,7 +374,11 @@ export class AmpStoryShoppingAttachment extends AMP.BaseElement {
               class="i-amphtml-amp-story-shopping-pdp-details-text"
               aria-hidden="true"
             >
-              {activeProductData.productDetails}
+              {activeProductData.productDetails
+                // Removes spaces after \n.
```

Comment:
I think you can replace both with:

```
    .replace(/\n\s*\n/g, '\n\n')
```

i.e. Replace two newlines (even if they have 0 or more whitespace characters between them) with just two newlines.
https://jsfiddle.net/o2x45vrj/

Reply:
🧠